src/features/projectsV2/ProjectsContext.tsx

```diff
@@ -88,8 +150,28 @@ export const ProjectsProvider: FC<ProjectsProviderProps> = ({
       projects,
       isLoading,
       isError,
+      filteredProjects,
+      debouncedSearchValue,
+      setDebouncedSearchValue,
+      isSearching,
+      setIsSearching,
+      topProjects,
+      tabSelected,
+      setTabSelected,
```

Comment:
Can tab selection state be moved to ProjectsSection instead? It isn't really needed within `ProjectsMap` as we don't show 2 different lists, or provide a way to move between tabs over there.